README.md

```diff
@@ -1,463 +1,161 @@
+#### Setup:
 [librelingo-yaml-loader](https://pypi.org/project/librelingo-yaml-loader/) can be installed using pip:
-
 ```sh
 pip install librelingo-yaml-loader
 ```
-
-#### Loading courses in a Python program
-
-You can use librelingo-yaml-loader to load YAML-based LibreLingo courses in your
-Python program.
-
+#### Loading courses in a Python program:
+You can use librelingo-yaml-loader to load YAML-based LibreLingo courses in your Python program.
 ```python
 from librelingo_yaml_loader import yaml_loader
 
-course = yaml_loader.load_course("./path/to/my/course")
+course = yaml_loader.load_course("./path/to/my/course") 
 ```
+`load_course` returns a [Course()](https://github.com/kantord/LibreLingo/blob/main/apps/librelingo_yaml_loader/librelingo_yaml_loader/data_types.py) object.
 
-`load_course` returns a [Course()](https://github.com/kantord/LibreLingo/blob/main/apps/librelingo_yaml_loader/librelingo_yaml_loader/data_types.py) object
-
-</td>
-</tr>
-</tbody>
-</table>
-
-
-<table>
-<tbody>
-<thead>
-<tr>
-<td>
- 
- 
 ### librelingo-json-export
- 
-</td>
-</tr>
-</thead>
-<tr>
-<td>
-
-
-[librelingo-json-export](https://pypi.org/project/librelingo-json-export/) is a Python 🐍 package that contains modules that let you
-export LibreLingo courses into JSON files that the web app will read. This might be useful if you want to deploy
-YAML-based LibreLingo courses as a web app.
-
-#### Setup
+[librelingo-json-export](https://pypi.org/project/librelingo-json-export/) is a Python 🐍 package that contains modules that let's you export LibreLingo courses into JSON files that will be read by the web app. This might be useful if you want to deploy YAML-based LibreLingo courses as a web app.
+#### Setup:
 [librelingo-json-export](https://pypi.org/project/librelingo-json-export/) can be installed using pip:
-
 ```sh
 pip install librelingo-json-export
 ```
-
-#### Exporting courses to JSON
-[librelingo-json-export](https://pypi.org/project/librelingo-json-export/) has a command-line program that can export YAML-based LibreLingo
-courses into the JSON format used by the web app.
-
-```
-Usage: cli.py [OPTIONS] INPUT_PATH OUTPUT_PATH
-
-  Convert a YAML course into a JSON course.
-
+#### Exporting courses to JSON:
+[librelingo-json-export](https://pypi.org/project/librelingo-json-export/) has a command line program that can export YAML-based LibreLingo courses into the JSON format used by the web app.
+```sh
+Usage:
+  cli.py [OPTIONS] INPUT_PATH OUTPUT_PATH
+    Convert a YAML course into a JSON course.
 Options:
-  --dry-run / --no-dry-run
-  --help                    Show this message and exit.
+  --dry-run / --no-dry-run --help
+    Show this message and exit.
 ```
 
-</td>
-</tr>
-</tbody>
-</table>
-
-
-
-
 
 ## Milestones
```

Comment:
perhaps this part is not important to keep this section. it's kinda old and it hasn't been kept up to date

Reply:
Okay I will update both readme and milestone here from now on